app/records/views.py

```diff
@@ -67,6 +76,33 @@ def record_detail_view(request, id):
         record=record,
     )
 
+    # Get the delivery options for the iaid
+    do_ctx = {}
+
+    try:
+        delivery_options = get_delivery_option(iaid=record.iaid)
```

Comment:
```suggestion
        api_result = get_delivery_option(iaid=record.iaid)
```
Suggested this name in line with comment for `doptions` to `api_result`.

Reply:
Agreed and changed